packages/server/modules/webhooks/services/webhooks.ts

```diff
@@ -0,0 +1,183 @@
+import { getServerInfo as getServerInfoFn } from '@/modules/core/services/generic'
+import { ForbiddenError } from '@/modules/shared/errors'
+import {
+  CountWebhooksByStreamId,
+  CreateWebhook,
+  CreateWebhookEvent,
+  DeleteWebhook,
+  GetWebhookById,
+  UpdateWebhook
+} from '@/modules/webhooks/domain/operations'
+import { Webhook } from '@/modules/webhooks/domain/types'
+import { SetValuesNullable } from '@speckle/shared'
+import crs from 'crypto-random-string'
+import {
+  StreamWithOptionalRole,
+  getStream as getStreamFn
+} from '@/modules/core/repositories/streams'
+import {
+  getUser as getUserFn,
+  UserWithOptionalRole
+} from '@/modules/core/repositories/users'
+import { Knex } from 'knex'
+
+const MAX_STREAM_WEBHOOKS = 100
+
+export const createWebhook =
```

Comment:
Its confusing that there's 2 operations, both called "create webhook". We sometimes have the same thing where there's the DB level creation operation and a service level operation that also adds extra validation on top (in this case this function probably makes more sense as a service level call). It just makes it more likely that someone coming in later will pick the wrong one and in the worst case scenario - avoid all validation.

I tend to solve this problem by forcing different naming. Maybe leaving the DB operation as "createX" and then having the service call be called "createValidatedX" or something like that. Ultimately you can name it all kinds of different things, but it should ideally have 1) its own operation type and 2) a different name.

Reply:
I changed the repository functions names since, I believe, in the repository context the name should be related to the table name which is `WebhookConfig`